.yamato/urp_postpro-osx-metal.yml

```diff
@@ -32,7 +32,8 @@ URP_PostPro_OSX_Metal_Standalone_mono_Linear_trunk:
       -  command: ssh -i ~/.ssh/id_rsa_macmini bokken@$BOKKEN_DEVICE_IP '$(python3 -m site --user-base)/bin/unity-downloader-cli --source-file ~/Graphics/unity_revision.txt -c editor -c il2cpp  --wait --published-only'
          retries: 2
       -  command: |5-
-                ssh -i ~/.ssh/id_rsa_macmini bokken@$BOKKEN_DEVICE_IP "export UPM_REGISTRY=https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates; ~/Graphics/utr  --artifacts_path=/Users/bokken/Graphics/TestProjects/UniversalGraphicsTest_PostPro/test-results --editor-location=.Editor --platform=StandaloneOSX --scripting-backend=Mono2x --suite=playmode --testfilter=$TEST_FILTER --testproject=/Users/bokken/Graphics/TestProjects/UniversalGraphicsTest_PostPro --timeout=2400 --zero-tests-are-ok=1 "
+                export GIT_REVISIONDATE=`git rev-parse HEAD | git show -s --format=%cI`
```

Comment:
Maybe it would be safer to also run a `PR on trunk` job, as it seems these changes impact all jobs?
I don't think they have any impact but just to be safe? Or do you feel the safety net is sufficient?

Reply:
im running a linux job to make sure the git command didnt affect anything, but the osx ones are the same as used in the perf job and that one worked :)